oadp-operator/controllers/velero.go

```diff
@@ -1,14 +1,486 @@
 package controllers
 
 import (
+	"fmt"
 	"github.com/go-logr/logr"
 	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"k8s.io/utils/pointer"
+)
+
+const (
+	VELERO        = "velero"
+	VELERO_NS     = "oadp-operator"
+	OADP_OPERATOR = "oadp-operator"
+	SERVER        = "server"
+	//TODO: Check for default secret names
+	VELERO_AWS_SECRET_NAME   = "cloud-credentials"
+	VELERO_AZURE_SECRET_NAME = "cloud-credentials-azure"
+	VELERO_GCP_SECRET_NAME   = "cloud-credentials-gcp"
+)
+
+// Environment Vars
+const (
+	LD_LIBRARY_PATH               = "LD_LIBRARY_PATH"
+	VELERO_NAMESPACE              = "VELERO_NAMESPACE"
```

Comment:
shouldn't this be "velero"?

Reply:
This is the env var key used on the container. The var name may want to communicate that